src/panel/direction/RouteSummaryInfo.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 
 import RouteVia from './RouteVia';
-import { Badge, Flex } from 'src/components/ui';
+import { Badge } from 'src/components/ui';
 import { formatDuration, formatDistance } from 'src/libs/route_utils';
 
 const RouteSummaryInfo = ({ route, vehicle }) =>
   <div>
-    <Flex alignItems="center" justifyContent="space-between" className="u-text--title">
+    <span className="u-text--title">
```

Comment:
This would be better as a div, so it doesn't depend on the next element to be on its own line